app/controllers/application.js

```diff
@@ -162,11 +162,11 @@ export default Controller.extend(cancelOrderMixin, {
       );
     },
 
-    checkout() {
+    async checkout() {
       if (this.get("cart.isEmpty")) {
         return;
       }
-
+      await this.get("cart").updateRequestedQuantity();
```

Comment:
not sure if want to await this,
`updateRequestedQuantity` is not returning any promise

Reply:
Yes , updateRequestedQuantity is returning promise , hence await is added